package.json

```diff
@@ -2385,6 +2385,207 @@
+                                                        "icon": "file",
+                                                        "type": "command",
+                                                        "command": "editor.action.formatDocument.multiple"
+                                                    },
+                                                    {
+                                                        "key": "S",
+                                                        "name": "Format selection with formatter",
+                                                        "icon": "selection",
+                                                        "type": "command",
+                                                        "command": "editor.action.formatSelection.multiple"
+                                                    }
+                                                ]
+                                            },
+                                            {
+                                                "key": "b",
+                                                "name": "+Backend",
+                                                "icon": "circuit-board",
+                                                "type": "bindings",
+                                                "bindings": [
+                                                    {
+                                                        "key": "d",
+                                                        "name": "Reset Database",
+                                                        "icon": "database",
+                                                        "type": "command",
+                                                        "command": "C_Cpp.ResetDatabase"
+                                                    },
+                                                    {
+                                                        "key": "w",
+                                                        "name": "Rescan Workspace",
+                                                        "icon": "project",
+                                                        "type": "command",
+                                                        "command": "C_Cpp.RescanWorkspace"
+                                                    }
+                                                ]
+                                            },
+                                            {
+                                                "key": "g",
+                                                "name": "+Go to",
+                                                "icon": "go-to-file",
+                                                "type": "bindings",
+                                                "bindings": [
+                                                    {
+                                                        "key": "a",
```

Comment:
Any reason why this is bound to `a`?

The c extension bound this to `alt + o`, and would it make more sense to consider find other files by using `f` or `o`?

Reply:
I see that makes sense. I guess we should just have the same keys with the cpp bindings.